### ISSUE TITLE
TSLint complains about semicolon on object spread operator

### ISSUE THREAD

**Issue by brunolm** [palantir/tslint#1943]
### Bug Report

- __TSLint version__: `4.2.0`
- __TypeScript version__: `2.1.4`
- __Running TSLint via__: Visual Studio Code (TSLint 0.7.1)

#### TypeScript code being linted

The variable is used. Without the spread it works.

```ts
    const something = { ...req.body };
    // [tslint] Unused variable: 'something' (no-unused-variable)
    // [tslint] Missing semicolon (semicolon)
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "class-name": true,
    "comment-format": [
      true,
      "check-space"
    ],
    "indent": [
      true,
      "spaces"
    ],
    "no-duplicate-variable": true,
    "no-eval": true,
    "no-internal-module": true,
    "no-trailing-whitespace": true,
    "no-var-keyword": true,
    "one-line": [
      true,
      "check-open-brace"
    ],
    "no-unused-variable": [
      true
    ],
    "quotemark": [
      true,
      "single"
    ],
    "semicolon": [
      true,
      "always"
    ],
    "triple-equals": [
      true,
      "allow-null-check"
    ],
    "typedef-whitespace": [
      true,
      {
        "call-signature": "nospace",
        "index-signature": "nospace",
        "parameter": "nospace",
        "property-declaration": "nospace",
        "variable-declaration": "nospace"
      }
    ],
    "variable-name": [
      true,
      "ban-keywords"
    ],
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type"
    ]
  }
}

```

#### Actual behavior

It thinks the variable is not being used and that a semicolon is missing.

#### Expected behavior

No errors/warnings

**Comment by andy-hanson** [comment]
This seems to work in `tslint@4.3.1` and `typescript@next`.

**Comment by brunolm** [comment]
Seems to be working on `tslint@4.3.1` and `typescript@2.1.5`
